coralproject/talk
# Scrolling to keep selected comments in view.

## What does this PR do?

When paging with keyboard shortcuts in the admin panel automatically scrolls the page to keep selected comments in view. 

## How do I test this PR?

1. Use 'j' and 'k' to page through the admin panel.
2. Page should scroll so that the selected comment is always visible.
